src/napari_imagej/setup_imagej.py

```diff
@@ -39,17 +39,17 @@ def imagej_init():
     )
     config.add_option(f"-Dimagej2.dir={os.getcwd()}")  # TEMP
     config.endpoints.append("io.scif:scifio:0.43.1")
+    config.endpoints.append("org.scijava:scijava-common:2.89.0")  # TEMP
```

Comment:
Is this necessary?

Reply:
[Let's find out](https://github.com/imagej/napari-imagej/runs/7983832447?check_suite_focus=true)!

Reply:
So, turns out it is. Does this have something to do with pom-scijava @ctrueden?

Reply:
It will only be necessary until we get `net.imagej:imagej:2.6.0` and `sc.fiji:fiji:2.6.0` released.